.github/workflows/l4lb.yaml

```diff
@@ -0,0 +1,69 @@
+name: Cilium L4LB XDP
```

Comment:
Also, the filename should be `test-l4lb.yaml` to be consistent with other tests

Reply:
I don't see any `test-*.yaml`:
```
$~/sandbox/gopath/src/github.com/cilium/cilium/.github/workflows [08:41:20]
> ls -a
.                   codeql-analysis.yml        go-check.yaml                       images.yaml        smoke-test-ipv6.yaml
..                  cyclonus-netpol-test.yaml  images-legacy-base-releases.yaml    kind-1.19.yaml     smoke-test.yaml
aks.yaml            docs.yaml                  images-legacy-hotfix-releases.yaml  l4lb.yaml
bpf-checks.yaml     eks.yaml                   images-legacy-releases.yaml         multicluster.yaml
build_commits.yaml  gke.yaml                   images-legacy.yaml                  nightly.yaml
```

Reply:
rebase and you shall see